examples/performance/00_overview.ipynb

```diff
@@ -852,7 +852,7 @@
       "  }\n",
       "  #pragma omp parallel num_threads(nthreads)\n",
       "  {\n",
-      "    const int tid = omp_get_thread_num();\n",
+      "    const int tid = omp_get_thread_num();;\n",
```

Comment:
I guess somewhere there is a redundant `;` printed

Reply:
yeah, I gave up on chasing these minor things

Reply:
it's probably some cgen misbehaviour